SpatialGDK/Source/SpatialGDK/Private/Utils/ComponentFactory.cpp

```diff
@@ -477,18 +479,20 @@ FWorkerComponentUpdate ComponentFactory::CreateHandoverComponentUpdate(Worker_Co
 	TArray<Schema_FieldId> ClearedIds;
 
 	uint32 BytesWritten = FillHandoverSchemaObject(ComponentObject, Object, Info, Changes, false, GetTraceKeyFromComponentObject(ComponentUpdate), &ClearedIds);
-	OutBytesWritten += BytesWritten;
 
 	for (Schema_FieldId Id : ClearedIds)
 	{
 		Schema_AddComponentUpdateClearedField(ComponentUpdate.schema_type, Id);
+		BytesWritten++; // Workaround so we don't drop updates that *only* contain cleared fields - JIRA UNR-3371
```

Comment:
Could we instead write a flag `DontDestroyUpdate` if we have `ClearedIds`

Reply:
BytesWritten is used outside this function to see if it should drop the update. This seemed like the smallest workaround until worker give us a fix.